asyncstdlib/_lrucache.py

```diff
@@ -72,6 +74,14 @@ def __wrapped__(self) -> AC:
         """The callable wrapped by this cache"""
         raise NotImplementedError
 
+    def __get__(
+        self: "LRUAsyncCallable[Any]", instance: object, owner: Optional[type] = None
```

Comment:
what do you think of using `from __future__ import annotations` so we can use the class as type hint for `self`

```suggestion
        self: LRUAsyncCallable[Any], instance: object, owner: Optional[type] = None
```

Reply:
Good point, I completely forgot that this is now covered by all supported versions. :+1: